### PR TITLE
feat: expand-preview

### LINKED ISSUE #146: Ability to expand preview to just show the preview on /editor

### PR DESCRIPTION
**Pull request by RohittCodes**
fixes #146 
image

### CODE REVIEW

**Review by imrishabh18 — COMMENTED** [review]
LGTM!!

File: src/components/PreviewContent.tsx [hunk 1]
```diff
@@ -195,6 +206,18 @@ export const PreviewContent = ({
                 </DropdownMenuContent>
               </DropdownMenu>
             </TabsList>
+            {onFullScreen && (
+              <Button
+                onClick={onFullScreen}
```
Comment (seveibar) [inline src/components/PreviewContent.tsx:211]:
```suggestion
                onClick={onToggleFullScreen}
```

**Review by seveibar — CHANGES_REQUESTED** [review]
minor name change, then good to go

**Review by seveibar — APPROVED** [review]
nice

**Pull request merged**